public/app/plugins/panel/canvas/editor/element/elementEditor.tsx

```diff
@@ -96,17 +96,27 @@ export function getElementEditor(opts: CanvasEditorOptions): NestedPanelOptions<
         layer.registerOptionsUI(builder, ctx);
       }
 
-      builder.addCustomEditor({
-        category: ['Layout'],
-        id: 'content',
-        path: '__', // not used
-        name: 'Quick placement',
-        editor: PlacementEditor,
-        settings: opts,
-      });
+      const shouldAddLayoutEditor = opts.element.item.standardEditorConfig?.layout ?? true;
+      if (shouldAddLayoutEditor) {
+        builder.addCustomEditor({
+          category: ['Layout'],
+          id: 'content',
+          path: '__', // not used
+          name: 'Quick placement',
+          editor: PlacementEditor,
+          settings: opts,
+        });
+      }
```

Comment:
```suggestion
       builder.addCustomEditor({
         category: ['Layout'],
          id: 'content',
          path: '__', // not used
          name: 'Quick placement',
          editor: PlacementEditor,
          settings: opts,
          showIf: () => shouldAddLayoutEditor,
        });
```

Reply:
I'm a little torn on modifying this to be slightly different way of showing / hiding just based on custom editor vs not 🤔 it might be more clear / maintainable to explicitly wrap in `if` statements. Thoughts?

Reply:
I think I'm comfortable leaving this as is to match with the other editors (just so when coming back to this in a while it is more explicit that it is shown / hidden based on the element settings 🤔 thoughts?

Reply:
yeah, should be fine!